addons/sale_loyalty/models/sale_order.py

```diff
@@ -29,6 +29,7 @@ class SaleOrder(models.Model):
     reward_amount = fields.Float(compute='_compute_reward_total')
 
     loyalty_data = fields.Json(compute='_compute_loyalty_data')
+    gift_card_count = fields.Integer(compute='_compute_gift_card_count')
```

Comment:
should be above `loyalty_data`  and may be add
`# Display Fields`
comment above them